### PR TITLE
fix #76611 ar.tommy.com

### LINKED ISSUE #76611: ar.tommy.com
### Issue URL (Incorrect Blocking)
 [https://ar.tommy.com/](https://adguardteam.github.io/AnonymousRedirect/redirect.html?url=https%3A%2F%2Far.tommy.com%2F)

### Screenshots
<details>
  <summary>Screenshot 1</summary>

  Screenshot 1
</details>
<details>
  <summary>Screenshot 2</summary>

  Screenshot 2
</details>

### System configuration

Information | value
--- | ---
AdGuard product: | AdGuard Browser extension v3.5.31
Browser: | Chrome
Filters: | <b>Ad Blocking:</b><br/> AdGuard Base<br/><br/><b>Privacy:</b><br/> AdGuard Tracking Protection<br/><br/><b>Annoyances:</b><br/> AdGuard Annoyances<br/><br/><b>Language-specific:</b><br/> AdGuard Turkish, <br/>AdGuard French, <br/>AdGuard Russian, <br/>AdGuard Chinese, <br/>AdGuard German, <br/>AdGuard Japanese, <br/>AdGuard Dutch, <br/>AdGuard Spanish/Portuguese

### PR DESCRIPTION
**Pull request by stanislav-atr**
#76611

### CODE REVIEW

File: AnnoyancesFilter/sections/cookies_specific.txt [hunk 1]
```diff
@@ -3520,6 +3520,7 @@ ticketea.com#$#html { overflow:visible!important; }
 fantrax.com##toast > section.toast.ng-trigger-enterLeave
 tommy.com#$#.ReactModalPortal { display:none!important; }
 tommy.com#$#body { overflow:visible!important; }
+tommy.com#$#.overlay--cookies { display: none !important; }
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/cookies_specific.txt:3523]: Why CSS? Scroll is not blocked. Old rules are obsolete, I guess.

File: AnnoyancesFilter/sections/cookies_whitelist.txt [hunk 2]
```diff
@@ -14,6 +14,9 @@ kupbilecik.pl#@#.cookie-container
 @@||app.usercentrics.eu/browser-ui/latest/bundle.js$domain=studycheck.de
 @@||api.usercentrics.eu/settings^$domain=studycheck.de
 @@||aggregator.service.usercentrics.eu/aggregate^
+! https://github.com/AdguardTeam/AdguardFilters/issues/76611
+!+ PLATFORM(ios, ext_safari)
+@@||tommy.com^$generichide
```
Comment (Alex-302) [inline AnnoyancesFilter/sections/cookies_whitelist.txt:19]: what the purpose of this rule?

File: AnnoyancesFilter/sections/cookies_specific.txt (hunk 1 above)
Comment (stanislav-atr) [reply AnnoyancesFilter/sections/cookies_specific.txt:3523]: There already exists
`ar.tommy.com#@$#body { overflow:visible!important; }`

File: AnnoyancesFilter/sections/cookies_whitelist.txt (hunk 2 above)
Comment (stanislav-atr) [reply AnnoyancesFilter/sections/cookies_whitelist.txt:19]: `#cookiesMessage` generic rule will remove element but modal backdrop will stay.

Comment (Alex-302) [reply AnnoyancesFilter/sections/cookies_whitelist.txt:19]: why not just these?
```
tommy.com#@##cookiesMessage
tommy.com#@#.cookies-message
```

**Review by Alex-302 — APPROVED** [review]

**Review by Alex-302 — APPROVED** [review]

**Pull request merged**